libmate-panel-applet/mate-panel-applet-factory.c

```diff
@@ -285,35 +278,41 @@ static const GDBusInterfaceVTable interface_vtable = {
 static GDBusNodeInfo *introspection_data = NULL;
 
 static void
-on_bus_acquired (GDBusConnection    *connection,
-		  const gchar        *name,
-		  MatePanelAppletFactory *factory)
+on_bus_acquired (GDBusConnection        *connection,
+                 const gchar            *name,
+                 MatePanelAppletFactory *factory)
 {
 	gchar  *object_path;
 	GError *error = NULL;
 
 	if (!introspection_data)
-		introspection_data = g_dbus_node_info_new_for_xml (introspection_xml, NULL);
-	object_path = g_strdup_printf (MATE_PANEL_APPLET_FACTORY_OBJECT_PATH, factory->factory_id);
+		introspection_data
+			= g_dbus_node_info_new_for_xml (introspection_xml, NULL);
+
+	object_path = g_strdup_printf (MATE_PANEL_APPLET_FACTORY_OBJECT_PATH,
+	                               factory->factory_id);
+
 	factory->connection = connection;
-	factory->registration_id = g_dbus_connection_register_object (connection,
-					   object_path,
-					   introspection_data->interfaces[0],
-					   &interface_vtable,
-					   factory, NULL,
-					   &error);
+	factory->registration_id
+		= g_dbus_connection_register_object (connection,
+                                                     object_path,
+                                                     introspection_data->interfaces[0],
+                                                     &interface_vtable,
+                                                     factory, NULL,
+                                                     &error);
```

Comment:
Doesn't align with `c of connection` line 297. Same when i open this commit in pluma (tabsize=8)

I think you will run into pain with mixed-usage of tabs and spaces all over this file.